File: datasets/IR-Plag/convertai-original-python-rest-java/case-04/plagiarized/L6/09/Level6.java
package IR/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
public class Level6 {

    public static void konversi(int mile) {
        if (mile != 11) {
            System.out.println(mile + "\t\t" + mile * 1.609);
            mile++;
            konversi(mile);
        }
    }
    
    public static void main(String[] args) {
        int mile = 1;
        System.out.println("Miles\t\tKilometers");
        System.out.println("-------------------------------");

        konversi(mile);
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L6/09/Level6.kt
package IR

/**
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level6 {
    @JvmStatic
    fun konversi(mile: Int) {
        if (mile != 11) {
            println("$mile\t\t${mile * 1.609}")
            konversi(mile + 1)
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")
        println("-------------------------------")
        konversi(1)
    }
}

